docs/quick-start.md

```diff
@@ -1,31 +1,52 @@
 # Quick Start
 
-To see how Argo Workflows work, you can install it and run examples of simple workflows and workflows that use artifacts.
+To see how Argo Workflows work, you can install it and run examples of simple workflows.
 
-Before you start you need a Kubernetes cluster and `kubectl` set-up
+Before you start you need a Kubernetes cluster and `kubectl` set-up to be able to access that cluster.
+
+⚠️ These instructions are intended to help you get started quickly. They are not suitable in production. For production installs, please refer to [the installation documentation](installation.md) ⚠️
 
 ## Install Argo Workflows
 
-To get started quickly, you can use the quick start manifest which will install Argo Workflows as well as some commonly used components:
+To install Argo Workflows, navigate to the [releases page](https://github.com/argoproj/argo-workflows/releases/latest) and find the release you wish to use (the latest full release is preferred).
+
+Scroll down to the `Controller and Server` section and execute the `kubectl` commands.
```

Comment:
I think we should include the commands here with a placeholder for the version number so the user doesn't have to nav away from this doc. What do you think about something like this?

```
kubectl create namespace argo
kubectl apply -n argo -f https://github.com/argoproj/argo-workflows/releases/download/v3.X.Y/install.yaml
```

Reply:
I did initially write it like that. Then came to the conclusion that some people will skim the page, then copy/paste the code and then cry a bit. What do you think?

Reply:
ha yeah, I see what you mean. Maybe something like the below and we leave it at 3.3.8 for now since that's stable and we let the community update it via PR in the future:

Below is an example of the install commands. Ensure you update the commend to install the correct version number.
```
kubectl create namespace argo
kubectl apply -n argo -f https://github.com/argoproj/argo-workflows/releases/download/v3.3.8/install.yaml

Reply:
I decided to go with your initial idea. I think it's better that an install completely fails (due to not reading) than the user ending up with a potentially old, unsupported version and not realising.